README.md

```diff
@@ -3,6 +3,17 @@
 An experiment with implementation of very simple operating system
 for [Malta](https://www.linux-mips.org/wiki/MIPS_Malta) board.
 
+Quickstart
+---
+
+To set up container-based local environment:
+
+```
+make -C docker run
```

Comment:
I'd also add build instruction here.
```suggestion
make -C docker build
make -C docker run
```

Reply:
@jpszczolowski right know the image is changing quite often, in longer term it is not supposed to be like that. That said, `docker run` will build the image if it does not exist, but it's not necessary to build it every time, as some people could deduce from this doc entry.

Reply:
Ok, I didn't know that `docker run` builds image if it doesn't exist

Reply:
@jpszczolowski just for your reference: https://github.com/cahirwpz/mimiker/pull/469/files#diff-072e81fe88d9b23fe00fc189279936a6R5 - this line is responsible for building the image

Reply:
@Tomatosoup97 `docker run` does not work inside clean repository, there is no generated `Dockerfile`